Monytor.Startup/Bootstrapper.cs

```diff
@@ -78,40 +78,35 @@ public class Bootstrapper {
             return builder;
         }
 
-        private static void SetupNotifications(ContainerBuilder builder, CollectorConfig collectorConfig) {
-            foreach (var notification in collectorConfig.Notifications) {
-                var b = ConfigCreator.LoadBehavior(typeof(NotificationBehavior<>), notification.GetType());
-                builder.RegisterType(b).Named(notification.Id, typeof(NotificationBehaviorBase));
-                builder.RegisterInstance(notification).Named(notification.Id, typeof(Notification));
-            }
-        }
-
-        private static void SetupVerfiers(ContainerBuilder builder, CollectorConfig collectorConfig) {
-            var distinctVerifiers = collectorConfig.Collectors
-                .SelectMany(x => x.Verifiers.Select(y => y.GetType())).Distinct();
+        private static void SetupNotifications(ContainerBuilder builder) {
+            var notifications = ImplementationTypeLoader.LoadAllConcreteTypesOf(typeof(Notification));
+            foreach (var notification in notifications) {
```

Comment:
Can we write it like this:
`ImplementationTypeLoader.LoadAllConcreteTypesOf<Notification>();`